go.mod

```diff
@@ -3,9 +3,12 @@ module github.com/nokia/CPU-Pooler
 go 1.13
 
 require (
+	github.com/Microsoft/go-winio v0.4.14 // indirect
```

Comment:
where are these dependencies coming from? they don't make much sense to me

Reply:
This PR is work in progress. Unit tests still fail in travis. Changed state to WIP. Sorry for the inconvenience. However, I am going to investigate this dependency.